### ISSUE TITLE
Opening the Morphic Tutorial can crash the image

### ISSUE THREAD

**Issue by codeZeilen** [hpi-swa-teaching/SWT18-Project-06#8]
When opening the Morphic Tutorial after checking it out from the repository (no prior Metacello installation, so probably some dependencies missing), the image halts. I have no further information at that point. :)

**Comment by codeZeilen** [comment]
Actually I do have the dependencies installed as they are in my image anyways. It might be another version but I do not think that there is a major version mismatch (but I didn't test it either).

**Comment by LeonMatthes** [comment]
What version of Squeak are you running at?
Did you not install Metacello at all or did you just not use it for the Motu installation?
I accidentally used a similar installation procedure without any issues:
Using a Squeak5.2alpha-18181-64Bit image, I did actually install Metacello to install Squot, but I did not install the MorphicTutorial via Metacello but just checked it out using Squot's Git Browser.
This installation procedure seems to work without any problems.

**Comment by codeZeilen** [comment]
Turns out this is related to a Pheno issue and does not originate in the Morphic Tutorial classes.

**Comment by codeZeilen** [comment]
Well it turns out not to be a Pheno issue but a misuse of the world:

```Smalltalk
MotuPlayground>>#initialize

	super initialize.
	self frameFractions: (0@0 rect: 1@1).
	self setUpBackground.
	super addMorph: Project currentWorld
```

**Comment by LeonMatthes** [comment]
This issue is fixed as of ad8a496eb7ce136a45ccc06b7177a3c6138c5d15
